src/lib/components/forms/DocumentUpload.svelte

```diff
@@ -122,6 +125,8 @@ progress through the three-part upload process.
 
   $: exceedsSizeLimit = files.some((file) => !isWithinSizeLimit(file));
 
+  $: disabled = !$user?.verified_journalist || loading || !csrf_token;
```

Comment:
I have been meaning to ship a fix to disable the button when there's nothing selected for upload. Could we sneak that in here?
```suggestion
  $: disabled = !$user?.verified_journalist || empty || loading || !csrf_token;
```

Reply:
For bonus points, a `disabledTooltip` value and a `<Tooltip>` around the button would help achieve our goal of giving users more direct feedback.